icu4c/source/tools/upropdump/Makefile.in

```diff
@@ -0,0 +1,97 @@
+## Makefile.in for ICU - tools/upropdump
+## Copyright (C) 2016 and later: Unicode, Inc. and others.
+## License & terms of use: http://www.unicode.org/copyright.html
+## Copyright (c) 1999-2011, International Business Machines Corporation and
+## others. All Rights Reserved.
+## Stephen F. Booth
+
+## Source directory information
+srcdir = @srcdir@
+top_srcdir = @top_srcdir@
+
+top_builddir = ../..
+
+include $(top_builddir)/icudefs.mk
+
+## Build directory information
+subdir = tools/upropdump
+
+TARGET_STUB_NAME = upropdump
```

Comment:
Optional: Most of our tools are named like verb+noun: generate-resource-bundles, make-conversion-files, ... maybe dumpuprops? writeuprops?

Reply:
In terms of prefixes, we have "icu" as in icuinfo, icupkg, icuswap, and "gen" as in genbrk, genccode, ...

In an effort to keep binary tools from polluting the global namespace, I went with "icuwriteuprops".

e28ef7ef007fe4e7c4023128ba28c7ee79f95b01